protractor.conf.js

```diff
@@ -2,7 +2,7 @@
 // https://github.com/angular/protractor/blob/master/docs/referenceConf.js
 
 /*global jasmine */
-var SpecReporter = require('jasmine-spec-reporter');
+var SpecReporter = require('jasmine-spec-reporter').SpecReporter;
```

Comment:
In  lathonez/clicker#205 they suggest
```javascript
let SpecReporter = require('jasmine-spec-reporter').SpecReporter;
```

But idea complains about `let` unsupported in this version of javascript. `yarn e2e` worked with `let` though.

Which one is correct? `var` or `let`?